README.md

```diff
@@ -33,10 +33,41 @@ All you have to do to get Accountant working is the following:
 (ns your-app-ns
   (:require [accountant.core :as accountant]))
 
-(accountant/configure-navigation!)
+(accountant/configure-navigation! {:nav-handler (fn [path] ...) :path-exists? (fn [path] ...)})
```

Comment:
If you're using secretary, there's exactly one way to do it. That's not true in bidi, because bidi is barely more than a pattern matching library. Also note than in bidi, I have to close over app-routes. There are other routers, like silk which are closer in approach to bidi than secretary (pure, functional, unopinionated).

Having to support both `:secretary` and the manual way is needless sugar, IMO. Also, you'd need a way to conditionally require secretary, so non-secretary users don't pull it into their CLJS builds.